core/templates/core/panel.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Panel Gwardii{% endblock %}
+
+{% load list_events %}
+
+{% block content %}
+    <div class="flex flex-row nowrap flex-grow h-full overflow-auto">
+        <div id="sidebar" class="hidden fixed top-0 md:relative md:block min-w-72 max-w-72 h-full bg-gray-800 shadow-2xl overflow-auto z-[0]">
+        <div class="flex flex-col gap-y-3 pb-28 md:pb-0" id="sidebar-content">
+            <a href="/" class="mx-4 py-3 my-2 h-12 max-w-full text-lg font-bold hover:underline">Strona Główna</a>
+            {% for namespace, links in sidebar_links.items %}
+            <div class="flex flex-grow flex-col divide-y divide-y-reverse divide-gray-600">
+                <h3 class="mx-4 py-3 my-2 h-12 max-w-full text-lg font-bold">{{ namespace }}</h3>
+                {% for link in links %}
+                <a class="mx-4 py-3 my-2 h-12 max-w-full rounded-xl hover:bg-slate-600 text-lg" href="{{ link.url }}" {% if link.new_tab %} target="_blank" {% endif %}>
+                    <i class="fa-regular fa-{{ link.icon }} mx-5 text-xl"></i><span>{{ link.name }}</span>
+                </a>
+                {% endfor %}
+            </div>
+            {% endfor %}
+        </div>
+        </div>
+        {% block panel_content %}
+        {% endblock %}
+    </div>
+    
+    <script>
+        const menuElement = document.querySelector("#sidebar")
+        menuElement.addEventListener("click", (event) => {
+            event.stopPropagation()
+        })
+        document.addEventListener('click', () => {
+            menuElement.classList.add("hidden")
+        })
+        document.querySelector("#mobile-menu-btn").addEventListener('click', (event) => {
+            event.stopPropagation()
+            menuElement.focus()
+            if (menuElement.classList.contains("hidden")) {
+                menuElement.classList.remove("hidden")
+            } else {
+                menuElement.classList.add("hidden")
+            }
+        })
+    </script>
+{% endblock %}
+
+{% block navbar_links %}
+    <li class="md:hidden">
+        <button id="mobile-menu-btn" type="button" class="inline-flex items-center justify-evenly p-2 mr-3 text-gray-300 focus:no-underline">
+            <span>Pasek boczny</span>
```

Comment:
Why?

Reply:
Plain bars could be confusing, that's why I've added this; Maybe it's not necessary?